content/news/articles/2025-07-01-rds-read-replicas-available.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: layouts/post
+tags: news
+title: Read replicas now available for RDS databases
+date: 2025-07-01
+excerpt: "Add a read replica for your RDS databases to handle read-only workloads"
+---
+
+The Cloud.gov team is excited to announce that adding a read replica is now supported for your databases on the platform.
+
+[A read replica is a read-only copy of your database which automatically replicates the data in your primary database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ReadRepl.html).
+
+Read replicas are useful for offloading tasks that are read intensive while your primary database handles create, update, and deletion of records. Isolating read operations to the read replica reduces the load on your primary database and also avoids database locking issues between read and write operations.
+
+Examples of read intensive tasks that may good use cases for a read replica include (but are not limited to):
+
+- Generating reports from your data
+- Serving data for API requests
+- Performing analytics queries
+
+## How to create a read replica
```

Comment:
I think we should drop this entire section except lines 35-41, which explain the scope of the feature. The news post should be an announcement and link out to our service docs, which should have all of these instructions. Thoughts?

Reply:
https://github.com/cloud-gov/site/pull/157/commits/b9ca2b59ef79d5d283392845bd2f7c42e8fa38fd